.github/workflows/ecosystem-ci-trigger.yml

```
@@ -9,7 +9,143 @@ jobs:
     runs-on: ubuntu-latest
     if: github.repository == 'vitejs/vite' && github.event.issue.pull_request && startsWith(github.event.comment.body, '/ecosystem-ci run')
     steps:
-      - uses: actions/github-script@v7
+      - name: Get PR Data
+        uses: actions/github-script@v7
+        id: get-pr-data
+        with:
+          script: |
+            console.log(`Get PR info: ${context.repo.owner}/${context.repo.repo}#${context.issue.number}`)
+            const { data: pr } = await github.rest.pulls.get({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              pull_number: context.issue.number
+            })
+            return {
+              num: context.issue.number,
+              branchName: pr.head.ref,
+              commit: pr.head.sha,
+              repo: pr.head.repo.full_name
+            }
+
+      - name: Check Package Existence
+        uses: actions/github-script@v7
+        id: check-package
+        with:
+          script: |
+            const prData = ${{ steps.get-pr-data.outputs.result }}
+            const url = `https://pkg.pr.new/vite@${prData.commit}`
+            const response = await fetch(url)
+            console.log(`Package check URL: ${url}, Status: ${response.status}`)
+
+            // Add 'rocket' reaction to the issue comment
+            if (response.status !== 404) {
```

```suggestion
            if (response.status === 404) {
```
Does the endpoint return 404 when the package exists? or is this condition inverted?